resallocserver/manager.py

```diff
@@ -611,6 +615,12 @@ def _assign_tickets(self):
                         if tag.priority is not None and tag.id in ticket_tags:
                             priority += tag.priority
 
+                    if resource.sandbox:
+                        # Re-used resources should be preferred to avoid
+                        # allocating new and new resources for the same
+                        # sandboxes.  TODO, make this configurable once needed.
```

Comment:
I guess the TODO would be better above the `REUSED_RESOURCE_PRIORITY` definition but that's just nitpicking.

Reply:
Indeed.

Reply:
Done, and merged.  Thank you!